src/lib/ngsi/ContextElementResponse.cpp

```diff
@@ -31,6 +31,14 @@
 #include "ngsi/ContextElementResponse.h"
 #include "rest/ConnectionInfo.h"
 
+/* ****************************************************************************
+*
+* ContextElementResponse::render -
+*/
+ContextElementResponse::ContextElementResponse()
+{
+  notPrune = true;
```

Comment:
This is the field I needed to create in ContextElementResponse to implement "shadow" entities that need to be prunned at the end if no CPrs is found for them. Although you also use this class, I think is has no effect for you.